h2o-core/build.gradle

```diff
@@ -36,7 +36,12 @@ dependencies {
   compile 'commons-lang:commons-lang:2.6'
   
   // Duke library: collection of String comparators
-  compile ('no.priv.garshol.duke:duke:1.2') { transitive = false }
```

Comment:
Is the transitive flag the cause of the SW build issues?

Reply:
It's not causing build issues, but the dependencies can't be used in Databricks cloud because of this. Even h2o-core as it is now out can't be used in DBC when specified via it's maven coordinates - full explanation is on the JIRA